web/index.html

```diff
@@ -1192,7 +1192,7 @@ <h4 class="modal-title" id="switchRegistryModalLabel">Switch Netdata Registry Id
 
 	'disk': {
 		title: 'Disks',
-		info: undefined
+		info: 'All metrics, except space and inodes, are read from <code>/proc/diskstats</code>. The field numbers mentioned in the subsection refer to the field numbers in the <a href="https://www.kernel.org/doc/Documentation/iostats.txt">Kernel Documentation</a>. Space metrics are read via system call <code>statvfs()</code>'
```

Comment:
I added some stub text on that page.
Do you want to complete it or you want me to do it?